oximeter/db/src/client/dbwrite.rs

```diff
@@ -0,0 +1,266 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! Implementation of client methods that write to the ClickHouse database.
```

Comment:
I'm not sure! For reference, this file was just split out of `src/client.rs`, which was getting too unwieldy. I don't think there's anything new in it.

But I do think we might change how we insert data to make querying easier. One example is https://github.com/oxidecomputer/omicron/issues/5289, though that's not really OxQL specific. The idea in that issue is that, by only reporting changes when they occur, we're making it hard to know what time range to search for when asking for data. It's probably not a huge deal for those particular metrics, since the temporal resolution is low-enough that you can likely just fetch the whole timeseries without much trouble. That's the best example I can think of now, though I'm also curious if there are others!

Reply:
Ah got it. I had not read through enough of the PR yet to realize that this was moved code - and the question was more along the lines of why now for a bunch of new DB write code. Thanks for the insight.